server/app/lib/ov_config.js

```diff
@@ -151,10 +151,12 @@ export const ov_config = {
   twilio_msg_svc_sid: getConfig("twilio_msg_svc_sid", false, null),
   neo4j_encryption: getConfig("neo4j_encryption", false, "ON"),
   hubspot_api_key: getConfig("hubspot_api_key", false, ""),
+  tripler_search_name_boost: getConfig("tripler_search_name_boost", false, 1),
+  alloy_bypass_keyword: getConfig("alloy_bypass_keyword", false, ""),
```

Comment:
I think this one should stay down with the other Alloy settings, yeah?

Reply:
`tripler_search_name_boost` is for tripler search, not Alloy